### PR TITLE
added 0.6 container

### PR DESCRIPTION
**Pull request by ChrisMcKenzie**

### CODE REVIEW

**Comment by hairyhenderson** [comment]
+1 would be great to get this merged :)

File: 0.6/consul-server/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,5 @@
+FROM quay.io/thecontrolgroup/consul-agent:0.6
```
Comment (hairyhenderson) [inline 0.6/consul-server/Dockerfile]: Shouldn't this be `gliderlabs/consul-agent:0.6`?

File: 0.6/consul-agent/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,6 @@
+FROM quay.io/thecontrolgroup/consul:0.6
```
Comment (hairyhenderson) [inline 0.6/consul-agent/Dockerfile]: Shouldn't this be `gliderlabs/consul:0.6`?

**Comment by ChrisMcKenzie** [comment]
sorry about that go a bit anxious so i built it myself temporarily forgot that i pr from master and was updating this PR thanks for the merge!

**Pull request merged**